app/api/feed.py

```
@@ -0,0 +1,32 @@
+
+from app.core.dependencies import CurrentUserId, DBSession
+from app.schemas.base import ApiResponse, ok
+from app.schemas.feed import FeedDetailResponse, FeedItemResponse
+from app.services.feed import FeedService
+
+router = APIRouter(prefix="/feed", tags=["feed"])
+
+
+@router.get("")
+async def list_feed(
+    user_id: CurrentUserId,
```

Ruff (0.14.14)

[warning] Unused function argument: `user_id`

(ARG001)